bitmovin-analytics-datasource/src/components/FilterRow.tsx

```diff
@@ -11,8 +11,8 @@ import {
 
 type Props = {
   readonly isAdAnalytics: boolean;
-  readonly onQueryFilterChange: (newFilters: QueryFilter[]) => void;
-  readonly filters: QueryFilter[];
+  readonly onQueryFilterChange: (newFilters: InputQueryFilter[]) => void;
```

Comment:
I would be also ok with name `QueryFilter` instead of new `InputQueryFilter` , because it would reduce changes here in this PR, but also because you have documented with JsDoc comment , why is `value: string`, so it is up to you if you wnat to rename it to have `Input` prefix